src/pybind/mgr/dashboard/frontend/src/app/core/navigation/identity/identity.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { AuthService } from '../../../shared/api/auth.service';
+@Component({
+  selector: 'cd-identity',
+  templateUrl: './identity.component.html',
+  styleUrls: ['./identity.component.scss']
+})
+export class IdentityComponent implements OnInit {
+  username: string;
+  
+  constructor(
+    private authService: AuthService,
+    private router: Router
+  ) { 
+    this.username = localStorage.getItem("dashboard_username");
```

Comment:
You should write a test using `AuthService` to make sure that the tests will fail if `'dashboard_username'` is not longer available.

Reply:
Please use `AuthStorageService.getUsername()` to retrieve the username.

Reply:
Ok, I will check it and modify.